sys/include/net/gcoap.h

```diff
@@ -8,7 +8,7 @@
  */
 
 /**
- * @defgroup    net_gcoap  CoAP
+ * @defgroup    net_gcoap  Gcoap
```

Comment:
Please at least use a separate commit for this.

Reply:
I agree completely. Sorry for the oversight.